src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -200,7 +229,30 @@ int halide_hexagon_remote_power_hvx_on() {
 int halide_hexagon_remote_power_hvx_off() {
     power_ref_count--;
     if (power_ref_count == 0) {
-        HAP_power_request(0, 0, -1);
+        HAP_power_request_t request;
+
+        request.type = HAP_power_set_HVX;
+        request.hvx.power_up = FALSE;
+        int retval = HAP_power_set(NULL, &request);
```

Comment:
is mini-dm working on your device, i wonder if there is some output about HVX powering on and off emitted to the mini-dm log ?